web_src/js/features/repo-editor-commit.ts

```diff
@@ -0,0 +1,102 @@
+import $ from 'jquery';
```

Comment:
We use fomanticQuery instead of referencing jQuery directly. This helps prepare for removing jQuery in the future.